test/tools/uri_spec_runner.ts

```diff
@@ -69,7 +75,9 @@ export function executeUriValidationTest(
       new MongoClient(test.uri);
       expect.fail(`Expected "${test.uri}" to be invalid${test.valid ? ' because of warning' : ''}`);
     } catch (err) {
-      if (err instanceof MongoRuntimeError) {
+      if (err instanceof MongoAzureError) {
+        // Azure URI errors don't have an underlying cause.
```

Comment:
I am a bit surprised this isn't just `MongoInvalidArgumentError`, even though it relates to Azure it's not Azure's error, it is our validation.

Reply:
Changed to `MongoInvalidArgumentError`.